### PR TITLE
BRO 10 Cards 1/11

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/liberator_urzas_battlethopter.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Name:Liberator, Urza's Battlethopter
+ManaCost:3
+Types:Legendary Artifact Creature - Thopter
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/liberator_urzas_battlethopter.txt:3]: No dashes in types

File: forge-gui/res/cardsfolder/upcoming/liberator_urzas_battlethopter.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+PT:1/2
+K:Flash
+K:Flying
+S:Mode$ CastWithFlash | ValidCard$ Artifact,Card.Colorless | ValidSA$ Spell | Caster$ You | Description$ You may cast colorless spells and artifact spells as though they had flash.
+T:Mode$ SpellCast | ValidCard$ Card | CheckSVar$ X | SVarCompare$ GTY | NoResolvingCheck$ True | Execute$ TrigCounter | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | TriggerDescription$ Whenever you cast a spell, if the amount of mana spent to cast that spell is greater than CARDNAME power, put a +1/+1 counter on NICKNAME.
+SVar:TrigCounter:DB$ PutCounter | Defined$ Self | CounterType$ P1P1 | CounterNum$ 1
+SVar:X:Count$TriggeredManaSpent
+SVar:Y:Count$CardPower
+DeckHints:Type$Urza|Eldrazi|Ugin|Artificer
+DeckHints:Type$Artifact & Ability$Counters
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/liberator_urzas_battlethopter.txt:13]: Combine DeckHints

File: forge-gui/res/cardsfolder/upcoming/the_stone_brain.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+SVar:TargetOpp:DB$ ChoosePlayer | ValidTgts$ Player.Opponent | Choices$ ThisTargetedPlayer | Defined$ You | SubAbility$ ExileYard
+SVar:ExileYard:DB$ ChangeZone | ChangeType$ Card.NamedCard | Origin$ Graveyard | DefinedPlayer$ Targeted  | Chooser$ You | Destination$ Exile | ChangeNum$ 4 | Hidden$ True | SubAbility$ ExileHand | StackDescription$ Search target opponent's graveyard, hand, and library for any number of cards with that name and exile them. Then that player shuffles their library.
+SVar:ExileHand:DB$ ChangeZone | Origin$ Hand | Destination$ Exile | DefinedPlayer$ Targeted  | ChangeType$ Card.NamedCard | ChangeNum$ 4 | Chooser$ You | SubAbility$ ExileLib | StackDescription$ None
+SVar:ExileLib:DB$ ChangeZone | Origin$ Library | Destination$ Exile | DefinedPlayer$ Targeted  | ChangeType$ Card.NamedCard | ChangeNum$ 4 | Chooser$ You | Shuffle$ True | StackDescription$ None
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/the_stone_brain.txt:8]: Where's the drawing?

File: forge-gui/res/cardsfolder/upcoming/urzas_command.txt [hunk 4]
```diff
@@ -0,0 +1,12 @@
+Name:Urza's Command
+ManaCost:2 U U
+Types:Instant
+A:SP$ Charm | Choices$ DBPump,DBPowerStone,DBKarnstruct,DBDraw | CharmNum$ 2
+SVar:DBPump:DB$ PumpAll | ValidCards$ Card.Creature+YouDontCtrl | NumAtt$ -2 | SpellDescription$ Creatures you don't control get -2/-0 until end of turn.
+SVar:DBPowerStone:DB$ Token | TokenTapped$ True | TokenScript$ c_a_powerstone | SpellDescription$ Create a tapped Powerstone token.
+SVar:DBKarnstruct:DB$ Token |  TokenScript$ c_0_0_a_construct_total_artifacts | SpellDescription$ Create a 0/0 colorless Construct artifact creature token with "This creature gets +1/+1 for each artifact you control."
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/urzas_command.txt:7]: Oracle says this tapped too?

File: forge-gui/res/cardsfolder/upcoming/urzas_command.txt [hunk 5]
```diff
@@ -0,0 +1,12 @@
+Name:Urza's Command
+ManaCost:2 U U
+Types:Instant
+A:SP$ Charm | Choices$ DBPump,DBPowerStone,DBKarnstruct,DBDraw | CharmNum$ 2
+SVar:DBPump:DB$ PumpAll | ValidCards$ Card.Creature+YouDontCtrl | NumAtt$ -2 | SpellDescription$ Creatures you don't control get -2/-0 until end of turn.
+SVar:DBPowerStone:DB$ Token | TokenTapped$ True | TokenScript$ c_a_powerstone | SpellDescription$ Create a tapped Powerstone token.
+SVar:DBKarnstruct:DB$ Token |  TokenScript$ c_0_0_a_construct_total_artifacts | TokenTapped$ True | SpellDescription$ Create a tapped 0/0 colorless Construct artifct creature token with "This creature gets +1/+1 for each artifact you control.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/urzas_command.txt:7]: Typo

**Comment by Simisays** [comment]
Thanks for all your comments as usual @tool4ever , though after some testing it seems that SVar:X:Count$TriggeredManaSpent
doesn't seem to work with Liberator, Urza's Battlethopter. Any thoughts ?

**Comment by tool4ever** [comment]
Right, you can't use triggeringObjects before it fully triggered

Try this instead: `ValidSA$ Spell.ManaSpent GTY`

File: forge-gui/res/cardsfolder/upcoming/urzass_sylex.txt [hunk 6]
```diff
@@ -0,0 +1,12 @@
+Name:Urzas's Sylex
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/urzass_sylex.txt:1]: name is `Urza's Sylex`

**Comment by Simisays** [comment]
Removed Urza's Liberator since I can't get the trigger to work

**Comment by tool4ever** [comment]
> Removed Urza's Liberator since I can't get the trigger to work

Seems to work fine for me?

image

**Comment by Simisays** [comment]
Hmm I'm casting Lighting bolt and getting the trigger while Liberator has power 4. I'm using | ValidCard$ Card | ValidSA$ Spell.ManaSpent GTX |
image

**Comment by tool4ever** [comment]
Right, good observation

Gonna push a fix on master now

**Review by tool4ever — APPROVED** [review]

**Pull request merged**